lib/public/HtmlChecker.js

```diff
@@ -12,24 +12,25 @@ function HtmlChecker(options, handlers)
 {
 	var linkEnqueued;
 	var thisObj = this;
-	
+
 	this.active = false;
 	this.baseUrl = undefined;
 	this.handlers = handlers || {};
 	this.linksSkipped = 0;
 	this.options = parseOptions(options);
 	this.parsed = false;
-	
+
 	this.htmlLinkParser = new HtmlLinkParser(this.options,
 	{
 		link: function(link)
 		{
 			// TODO :: if link.url.resolved results in null, this function will run AGAIN in checkUrl()
 			linkObj.resolve(link, thisObj.baseUrl, thisObj.options);
-			
+
 			// If link should be ignored
 			if (
 			   	(link.internal===true && thisObj.options.excludeInternalLinks===true) ||
+					(link.internal===false && thisObj.options.excludeExternalLinks===true) ||
```

Comment:
Check the white-space here. I use tabs up to the `if`, and spaces after so that regardless of tab size, it always aligns.